app/models/attachment.rb

```diff
@@ -7,7 +7,7 @@ class Attachment
   field :file_id, type: String
   field :file_url, type: String
 
-  embedded_in :specialist_document_edition
+  embedded_in :section_edition
```

Comment:
I think it would be useful to mention why this change does not require any database migration, i.e. can you confirm that none of the existing attachments store a reference to their parent edition using this key. I'm guessing it's because this is an embedded document...?